packages/installer/cos.sh

```diff
@@ -4,17 +4,16 @@ set -e
 PROG=$0
 
 ## Installer
-PROGS="dd curl mkfs.ext4 mkfs.vfat fatlabel parted partprobe grub2-install grub2-editenv"
-DISTRO=/run/rootfsbase
-ISOMNT=/run/initramfs/live
-ISOBOOT=${ISOMNT}/boot
-TARGET=/run/cos/target
-RECOVERYDIR=/run/cos/recovery
-RECOVERYSQUASHFS=${ISOMNT}/recovery.squashfs
-GRUBCONF=/etc/cos/grub.cfg
+_PROGS="dd curl mkfs.ext4 mkfs.vfat fatlabel parted partprobe grub2-install grub2-editenv"
+_DISTRO=/run/rootfsbase
+_ISOMNT=/run/initramfs/live
+_TARGET=/run/cos/target
+_RECOVERYDIR=/run/cos/recovery
+_RECOVERYSQUASHFS=${_ISOMNT}/recovery.squashfs
+_GRUBCONF=/etc/cos/grub.cfg
```

Comment:
This one is configurable by a user. Shall we maybe prefix all of the ones that are configurable?

For backward compatibility we should also respect `GRUBCONF` in this case too.

Reply:
That is true.

My idea is to move the configurable ones to be set on the `load_config` and use only internal ones. So no touching any external ones or using them at all, so they only entrypoints for the script are the switches or the supported configs mentioned under https://rancher-sandbox.github.io/cos-toolkit-docs/docs/customizing/general_configuration/

Reply:
should be fixed by the change below